scripts/release/release.sh

```diff
@@ -259,7 +259,8 @@ verify()
     fi
   fi
   if [ -s "$2/$BOWER_JSON" ]; then
-    bower install $2/$BOWER_JSON
+    cp $2/$BOWER_JSON .
+    bower install
```

Comment:
Can you please add a line to remove the file after the install? Not a problem with Travis, but running the script manually will leave a bower.json file hanging around locally.